Tests/Unit/Util/PostsUtilTest.php

```diff
@@ -22,7 +21,7 @@ public function dataProviderSlugify()
             array('this is a slug', 'this-is-a-slug'),
             array('This Is A SLUG', 'this-is-a-slug'),
             array('je suis une dévelopeur web', 'je-suis-une-developeur-web'),
-            array('foo!"  812391 !"£ %!"$£*%!"$£', 'foo-812391'),
+            array('foo!"  812391 !"£ %!"$£*%!"$£', 'foo-812391-lb-lblb'),
```

Comment:
i did this change .. I suspect that `` !"£ %!"$£*%!"$£`` was previously ignored and stripped by the slugifyer and is now transliterated to.